src/Raven.Server/Config/Categories/ServerConfiguration.cs

```diff
@@ -56,5 +56,27 @@ public class ServerConfiguration : ConfigurationCategory
         [TimeUnit(TimeUnit.Minutes)]
         [ConfigurationEntry("Server.CpuCredits.ExhaustionBackupDelayInMin", ConfigurationEntryScope.ServerWideOnly)]
         public TimeSetting CpuCreditsExhaustionBackupDelay { get; set; }
+
+        [Description("EXPERT: A command or executable that will provide RavenDB with the current CPU credits balance.")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Server.CpuCredits.Exec", ConfigurationEntryScope.ServerWideOnly)]
+        public string CpuCreditsExec { get; set; }
+
+        [Description("EXPERT: The command line arguments for the Server.CpuCredits.Exec command or executable.")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Server.CpuCredits.Exec.Arguments", ConfigurationEntryScope.ServerWideOnly)]
+        public string CpuCreditsExecArguments { get; set; }
+
+        [Description("EXPERT: The number of seconds between every invokation of the Cpu Credits executable. Default: 30 minutes.")]
+        [DefaultValue(30*60)]
+        [TimeUnit(TimeUnit.Seconds)]
```

Comment:
Why not in Minutes?

Better to write in description then 1800 seconds to avoid unit change

Reply:
Because in the code it is used as seconds. Do you prefer I use minutes here... and in the code do a conversion?

Reply:
Oh ok... I got what you mean now. You prefer that in the text I'll write "Default: 1800 seconds"

Reply:
Let's change this to Minutes